Demo/Demo/FeatureViewControllers/FeatureBaseViewController.swift

```diff
@@ -87,49 +97,18 @@ class FeatureBaseViewController: UIViewController {
         DemoMerchantAPI.sharedService.createOrder(orderParams: orderRequestParams) { result in
             switch result {
             case .success(let order):
-                self.updateTitle("Order ID: \(order.id)")
-                self.orderID = order.id
+                self.baseViewModel.updateTitle("Order ID: \(order.id)")
+                DispatchQueue.main.async {
+                    self.baseViewModel.orderID = order.id
```

Comment:
Should updating the order ID always be on the main thread or just in this case? If so would it make more sense to have an updateOrderID method that calls `DispatchQueue.main.async` under the hood (similar to updateTitle)?

Reply:
I think it makes sense to always be on the main thread - SwiftUI needs the update to the `@Published var` to happen on the main thread which was why this was added. Updating this in a similar manner to the `updateTitle` function.

Reply:
It would be fine for this to always happen on the main thread - this was originally updated because of how SwiftUI handles the `@Published var` of the order ID. I think it makes since to format this in a similar way to the `updateTitle` function.